packages/blade/src/components/Tabs/_decisions/decisions.md

```diff
@@ -0,0 +1,385 @@
+    <Text>Tab Panel Content For Disputes</Text>
+  </Tab>
+</Tabs>
+```
+
+We decided not to go with this API because of these downsides: 
+
+#### 1. Quite tricky/hack to implement
+
+With the API that you proposed it will be quite tricky to map the JSX to the HTML structure that a typical tab requires. 
+
+A tab's HTML structure is as follows
+
+```html
+<div>
+  <div role="tablist">
+    <button role="tab">tab item 1</button>
+    <button role="tab">tab item 2</button>
+    <button role="tab">tab item 3</button>
+  </div>
+  
+  <div>
+    <section role="tabpanel"></section>
+    <section role="tabpanel"></section>
+    <section role="tabpanel"></section>
+  </div>
+</div>
+```
+
+We need to achieve the above HTML structure, and to do this with the API that you proposed will get very hacky since now we need to use React.cloneElement and save those JSX nodes in state and render them in appropriate place. 
+
+```html
+<Tabs>
+  // this will be the button with role="tab", need to React.cloneElement and move it to the right place, and while placing also need to make sure it has a parent of role="tablist"
+  <Tab value={} leading={} trailing={} title={}>  
+    // this is role="tabpanel" need to React.cloneElement and move it to the right place
+    {...the jsx content for the content of the tab} 
+  </Tab>
+</Tabs>
+```
+
+#### 2. The JSX doesn't map well with the visual hierarchy / the HTML structure
+
+If you notice the HTML structure pointed out in point #1 you'll see that it's 1:1 with the JSX API that i'm proposing. 
+
+The current API that is proposed also maps 1:1 with the visual hierarchy of the component, but the second one doesn't map very well.  
+
+![Frame 1 (11)](https://github.com/razorpay/blade/assets/35374649/9058d53f-aa5b-4a45-9111-5bfbdafa8f27)
+
+#### 3. Extensibility
+
+The current API is also much more easier to extend in case any future usecase arises, for example let's say we want to use these tabs like a segmented control (which works like radio buttons but doesn't associate with any "tabpanels" we can simply remove the tabpanels and change the underlying html structure to accommodate for any future usecases. 
+
+Check how reshaped is allowing this [extensibility](https://reshaped.so/content/docs/components/tabs#using-in-forms) - if you pass `name` prop it will convert the tabs to work like a Radio. 
+
+#### 4. API Familiarity 
+
+The current API that is proposed is also much more familiar and is similar to how many other design systems are approaching compared to the second one: 
+
+- [Reshaped](https://reshaped.so/content/docs/components/tabs)
+- [Chakra](https://chakra-ui.com/docs/components/tabs/usage)
+- [MUI](https://mui.com/material-ui/react-tabs/#experimental-api)
+- [Atlassian](https://atlassian.design/components/tabs/examples)
+- [RadixUI](https://www.radix-ui.com/primitives/docs/components/tabs)
+- [HeadlessUI](https://headlessui.com/react/tabs)
+
+
+## Motion
```

Comment:
put the motion video of tabs switching for both vertical+horizontal orientation and bordered and filled variant

Reply:
There are many variants, attached the motion prototype instead. People can open the prototype and inspect the motion.

Reply:
can you record a tiny video from the prototype and it as a sample here? want it to act as a cue for readers to open the prorotype

Reply:
@anuraghazra bumping this up. link will easy be missed and rarely people will actually open it

Reply:
Added